src/main/java/org/betonquest/betonquest/objectives/BlockObjective.java

```diff
@@ -36,6 +38,25 @@ public BlockObjective(final Instruction instruction) throws InstructionParseExce
         noSafety = instruction.hasArgument("noSafety");
     }
 
+    @Override
+    public String getProperty(final String name, final Profile profile) {
+        switch (name.toLowerCase(Locale.ROOT)) {
+            case "amount":
+            case "left":
+            case "total":
+                return super.getProperty(name, profile);
+            case "absoluteamount":
+            case "absoluteleft":
+            case "absolutetotal":
+                return Integer.toString(
+                        Math.abs(
+                                Integer.valueOf(
+                                        super.getProperty(name.replace("absolute", ""), profile))));
```

Comment:
That's a creative solution 😂 
Given the simplicity of this solution I would recommend adding the absolute variants of the variables to the counting objective class instead. Then all objectives supporting counting objective support this feature.

```JAVA
    public String getProperty(final String name, final Profile profile) {
        final Integer data = switch (name.toLowerCase(Locale.ROOT)) {
            case "amount" -> getCountingData(profile).getCompletedAmount();
            case "left" -> getCountingData(profile).getAmountLeft();
            case "total" -> getCountingData(profile).getTargetAmount();
            case "absoluteamount" -> Math.abs(getCountingData(profile).getCompletedAmount());
            case "absoluteleft" -> Math.abs(getCountingData(profile).getAmountLeft());
            case "absolutetotal" -> Math.abs(getCountingData(profile).getTargetAmount());
            default -> null;
        };
        return data == null ? "" : data.toString();
    }
````

Reply:
I just followed the description of the issue that only BlockObjectives should have those properties.

Will change it according to your suggestion.